packages/block-editor/src/hooks/layout.scss

```diff
@@ -1,23 +1,13 @@
-.block-editor-hooks__layout-controls-units {
-	display: flex;
-	flex-direction: column;
-	gap: $grid-unit-20;
-}
-
-.block-editor-block-inspector .block-editor-hooks__layout-controls-unit-input {
-	margin-bottom: 0;
-}
-
-.block-editor-hooks__layout-controls-reset {
-	display: flex;
-	justify-content: flex-end;
-	margin-bottom: $grid-unit-30;
+.block-editor-hooks__layout-constrained {
+	.components-base-control {
+		margin-bottom: 0; // Cancel out margins added by block inspector
+	}
 }
 
-.block-editor-hooks__layout-controls-helptext {
+.block-editor-hooks__layout-constrained-helptext {
```

Comment:
Renamed the class for consistency with the JS file name. (Naming it `controls` seems too broad to avoid collisions.)